[2025-02-12T23:46:00.906+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T23:46:00.921+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_omdb_api.teste scheduled__2025-02-12T23:45:00+00:00 [queued]>
[2025-02-12T23:46:00.926+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_omdb_api.teste scheduled__2025-02-12T23:45:00+00:00 [queued]>
[2025-02-12T23:46:00.927+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-12T23:46:00.937+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): teste> on 2025-02-12 23:45:00+00:00
[2025-02-12T23:46:00.943+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=445) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-12T23:46:00.944+0000] {standard_task_runner.py:72} INFO - Started process 446 to run task
[2025-02-12T23:46:00.944+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_omdb_api', 'teste', 'scheduled__2025-02-12T23:45:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/call_api.py', '--cfg-path', '/tmp/tmpz7hpin3a']
[2025-02-12T23:46:00.945+0000] {standard_task_runner.py:105} INFO - Job 24: Subtask teste
[2025-02-12T23:46:00.982+0000] {task_command.py:467} INFO - Running <TaskInstance: call_omdb_api.teste scheduled__2025-02-12T23:45:00+00:00 [running]> on host 76fc25d1410f
[2025-02-12T23:46:01.062+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Celso Cazetto' AIRFLOW_CTX_DAG_ID='call_omdb_api' AIRFLOW_CTX_TASK_ID='teste' AIRFLOW_CTX_EXECUTION_DATE='2025-02-12T23:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-12T23:45:00+00:00'
[2025-02-12T23:46:01.063+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-12T23:46:01.074+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-12T23:46:01.075+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'source /home/***/venv/omdb_env/bin/activate && python /opt/***/pipeline/main.py']
[2025-02-12T23:46:01.083+0000] {subprocess.py:99} INFO - Output:
[2025-02-12T23:46:01.923+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:01,922 - INFO - ESTOU RODANDO ESSA PORRA
[2025-02-12T23:46:01.924+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:01,923 - INFO - Carregando o arquivo de configuração
[2025-02-12T23:46:01.924+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:01,924 - INFO - Buscando os dados de filme do ID:tt0371746
[2025-02-12T23:46:02.032+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:02,032 - INFO - Dados da API carregados com sucesso
[2025-02-12T23:46:02.033+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:02,032 - INFO - Iniciando transformação dos dados.
[2025-02-12T23:46:02.040+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:02,040 - INFO - Estabelecendo conexão com o banco de dados:
[2025-02-12T23:46:02.069+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:02,069 - INFO - Conexão estabelecida com sucesso.
[2025-02-12T23:46:02.087+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:02,087 - INFO - Iniciando o inserção na tabela: omdb.filmes
[2025-02-12T23:46:02.090+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:02,090 - INFO - 0 Inseridos na tabela omdb.filmes
[2025-02-12T23:46:02.094+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:02,094 - INFO - Iniciando o inserção na tabela: omdb.genero
[2025-02-12T23:46:02.095+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:02,095 - INFO - 0 Inseridos na tabela omdb.genero
[2025-02-12T23:46:02.103+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:02,103 - INFO - Iniciando o inserção na tabela: omdb.filme_genero
[2025-02-12T23:46:02.104+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:02,104 - INFO - 0 Inseridos na tabela omdb.filme_genero
[2025-02-12T23:46:02.106+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:02,106 - INFO - Conexão finalizada com o banco
[2025-02-12T23:46:02.107+0000] {subprocess.py:106} INFO - 2025-02-12 23:46:02,106 - INFO - Fim do processo de ingestão.
[2025-02-12T23:46:02.107+0000] {subprocess.py:106} INFO - 2.0.38
[2025-02-12T23:46:02.185+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-12T23:46:02.205+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-12T23:46:02.206+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=call_omdb_api, task_id=teste, run_id=scheduled__2025-02-12T23:45:00+00:00, execution_date=20250212T234500, start_date=20250212T234600, end_date=20250212T234602
[2025-02-12T23:46:02.242+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T23:46:02.254+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T23:46:02.256+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
